invenio_accounts_rest/views.py

```diff
@@ -41,15 +41,20 @@
 from werkzeug.local import LocalProxy
 
 from invenio_accounts_rest.errors import MaxResultWindowRESTError
-from invenio_accounts_rest.loaders import default_json_loader_with_profile, \
-    default_json_loader_without_profile, \
-    default_json_patch_loader_with_profile, \
-    default_json_patch_loader_without_profile
+from invenio_accounts_rest.loaders import (
```

Comment:
The bug I had is that isort was always putting the first imported function on the import line like this:
```python
from invenio_accounts_rest.loaders import default_account_json_loader_with_profile \
...
```
This breaks the 80 column rule.
I tried with the same isort configuration we use in other modules and also tried to set the `line_length` parameter to < 79 values, but this didn't change anything.
This bug might be related to timothycrosley/isort#416. The PR is merged but yet not released.

Once this is fixed in isort we can just run it again with the default config and it should fix all the imports.